README.md

```diff
@@ -8,10 +8,22 @@ This extension currently supports:
 
 If you're new to CMS Hub, check out how to [get started with local development.](https://designers.hubspot.com/docs/tools/local-development)
 
+## Getting Started
+This extension introduces new file languages: `HTML + HUBL` and `HTML + CSS`. In order to use these new features, you'll want to make sure you've told VSCode to use these langauges. For one-off files, you can simply change the language in the lower right hand corner of your screen. However, we recommend adjusting your file associations under your User or Workspace preferences:
```

Comment:
...under your User or Workspace preferences... so HubL syntax highlighting of will automatically be configured for your projects.

?